src/js/components/TabForm.js

```diff
@@ -161,7 +161,7 @@ class TabForm extends mixin(InternalStorageMixin) {
     let formRowClass = this.props.formRowClass;
 
     let panels = Object.keys(multipleDefinition).map((formKey, i) => {
-      let panelClassSet = classNames('form', {
+      let formPanelClassSet = classNames('form-panel', {
         'hidden': currentTab !== formKey
```

Comment:
With this change, we're hding the whole `form-panel` element instead of its child.